plugin-build/src/test/resources/testFixtures/appTestProject/gradle.properties

```diff
@@ -1,6 +1,5 @@
 org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=1g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC -XX:+CrashOnOutOfMemoryError
 org.gradle.daemon=false
-org.gradle.caching=true
```

Comment:
disabled this because there's no really noticable gain in build speed, but the tests might be non-deterministic with the caching flag on